src/Components/Inputs/Number/UpNumber.tsx

```diff
@@ -95,7 +95,8 @@ applyDecimalPlace = (value: string) =>{
 
 handleNumericChange = (event: React.ChangeEvent<any>, valueAsString: string) => {
  if ( this.isValueMatched(valueAsString) ) {
-     this.handleChangeEvent(event, valueAsString);
+     const currentValue = valueAsString.replace(',','.');
+     return parseFloat(currentValue) > this.props.max ? null : this.handleChangeEvent(event, valueAsString);
```

Comment:
this.props.max && parseFloat(currentValue) > this.props.max ?

Reply:
Même chose pour min 
aussi parseFloat(currentValue) > this.props.max ? null  => parseFloat(currentValue) > this.props.max ? this.props.max ?